pkg/controller/podautoscaler/config/v1alpha1/defaults.go

```diff
@@ -34,6 +34,9 @@ import (
 // run it in your wrapper struct of this type in its `SetDefaults_` method.
 func RecommendedDefaultHPAControllerConfiguration(obj *kubectrlmgrconfigv1alpha1.HPAControllerConfiguration) {
 	zero := metav1.Duration{}
+	if obj.ConcurrentHorizontalPodAutoscalerSyncs == 0 {
+		obj.ConcurrentHorizontalPodAutoscalerSyncs = 1
```

Comment:
I set the default to 1 - to keep the current behaviour, though I'd rather increase the default value to 5 or 10. WDYT?

Reply:
As discussed on SIG autoscaling, increasing this to 5.